### DISCUSSION TITLE
Graphic problem with macbook air 4,2 running sonoma

### DISCUSSION THREAD

**Discussion by nicpin01** [community/community#87028] [Discussions]
### Select Topic Area

Question

### Body

Hi, I have a macbook air 4,2 running Sonoma with OCLP, post patches applied, there’s a graphic problem with the integrated screen, black screen with some white lines sometime. The mini display port allows me to have something on external screen, once I blindly entered my password. When I look at the graphic settings, integrated screen appears as if it were working.
I'm afraid I won't be able to go back to an earlier version or factory setting, because before the ios is started following password, nothing is displayed on the external screen too. 
Do anyone has similar problem and what can I do?
Thank you!

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @nicpin01 ! We're happy you're here, the Discussions category is solely related to conversations around the GitHub product Discussions.

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Apple product like the [Mac community](https://discussions.apple.com/community/mac). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
